Makefile

```diff
@@ -20,3 +22,7 @@ upload_package: format_package build_package
 # Conda env
 create_conda_env_file:
 	 conda env export > environment.yml
```

Comment:
can we have a pip option pls?

Reply:
rather, no pip option in makefile, but not sure that matters since it is in the `README.md`

Reply:
I have a pip option (but it doesn't work apparently). The problem is ray which has caused a headache from the start. Might look at this a bit more tonight.

The Makefile is only for building and deploying. It's not for users.